instr.ml

```diff
@@ -1,12 +1,6 @@
-module Variable = struct
-  type t = string
-  let compare = String.compare
-end
+module Variable = String
```

Comment:
Welcome :-) . Gabriel suggested this while I was unifying the freshen family of functions. He said that - "its dirty but its you who is doing it". I guess the joke's on him now ;-)